app-modules/service-management/src/Observers/ServiceRequestObserver.php

```diff
@@ -86,6 +86,8 @@ public function saved(ServiceRequest $serviceRequest): void
     {
         CreateServiceRequestHistory::dispatch($serviceRequest, $serviceRequest->getChanges(), $serviceRequest->getOriginal());
 
+        $serviceRequest->load('status');
```

Comment:
Did this actually solve the issue we discussed where you are getting the wrong status in the email? If not, let's remove it.

Reply:
HI,

Yes it's not solved that issue.

Thank you.